### PR TITLE
Modernize script

### PR DESCRIPTION
**Pull request by alkino**
A bit of cleaning for this script.
`CURRENT_DIR` is added for making script relocatable

### CODE REVIEW

**Review by pramodk — APPROVED** [review]
LGTM

**Review by alexsavulescu — APPROVED** [review]

File: runtests [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 export NRNUNIT_USE_LEGACY=1
 
-sh testutil/clean
-neurondemo -c 'quit()'
+CURRENT_DIR=$(dirname $0)
```
Comment (matz-e) [inline runtests:5]:
```suggestion
CURRENT_DIR=$(readlink $(dirname $0))
```
?

Comment (pramodk) [reply runtests:5]: not necessary to resolve path but why not...

**Comment by alkino** [comment]
`infrastructure` script start with `exit 0` ^^

**Comment by pramodk** [comment]
> infrastructure script start with exit 0 ^^

looking at the history, that seems very old, prototype...

**Pull request merged**